src/transformers/adapter_heads.py

```diff
@@ -335,15 +336,23 @@ def register_custom_head(self, identifier, head):
 
     @property
     def active_head(self):
-        return self._active_head
+        return self._active_heads[0] if len(self._active_heads) > 0 else None
 
     @active_head.setter
     def active_head(self, head_name):
-        self._active_head = head_name
+        self._active_heads = [head_name]
         if head_name is not None and head_name in self.heads:
             self.config.label2id = self.heads[head_name].config["label2id"]
             self.config.id2label = self.get_labels_dict(head_name)
 
+    # TODO: maybe think of something better. remove properties for single head? (breaks compability)
+    @property
+    def active_heads(self):
+        return self._active_heads
+
+    def set_active_heads(self, head_names: Sequence):
+        self._active_heads = head_names
+
```

Comment:
@hSterz For parallel inference, we need to be able to set multiple active heads. This is the simplest possible solution, but having both `active_head` and `active_heads` might be confusing. What do you think?

Reply:
I think we need to document what it means to set multiple active heads. There are two options how they could be utilized.

1. Each active head is applied on each input example, resulting in n_examples*K predictions (k being the number of heads)
2. The first head is applied to the first example, second to second ... etc.

Reply:
Added some doc to the property.

I also opted for keeping only the `active_head` property (singular) and allowing to pass either a single name or a list. This naming may be a bit weird if passing a list, but it doesn't introduce any further breaking changes to v1 and doesn't have the confusing singular/ plural split.